samples/README.md

```diff
@@ -123,7 +123,7 @@ This sample demonstrates converting a series of UltraHDR images into a HDR
 - [UltraHDR x OpenGLES SurfaceView](graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/opengl/UltraHDRWithOpenGL.kt):
 This sample demonstrates displaying an UltraHDR image via and OpenGL Pipeline 
 - [Video Composition using Media3 Transformer](media/video/src/main/java/com/example/platform/media/video/TransformerVideoComposition.kt):
-This sample demonstrates concatenation of two video assets using Media3 
+This sample demonstrates concatenation of two video assets and an image using
```

Comment:
nit: I think we've lost "Media3" at the end of this sentence?

Reply:
whoops! good catch. fixed.